x/wasm/keeper/msg_dispatcher.go

```diff
@@ -81,6 +82,14 @@ func (d MessageDispatcher) dispatchMsgWithGasLimit(ctx sdk.Context, contractAddr
 func (d MessageDispatcher) DispatchSubmessages(ctx sdk.Context, contractAddr sdk.AccAddress, ibcPort string, msgs []wasmvmtypes.SubMsg) ([]byte, error) {
 	var rsp []byte
 	for _, msg := range msgs {
+
+		// Prevent the use of fswap and fbridge module in Submessages
+		// https://github.com/Finschia/finschia-sdk/pull/1336, https://github.com/Finschia/finschia-sdk/pull/1340
+		if stargateMsg := msg.Msg.Stargate; stargateMsg != nil &&
+			(strings.Contains(stargateMsg.TypeURL, "lbm.fswap.v1") || strings.Contains(stargateMsg.TypeURL, "lbm.fbridge.v1")) {
+			return nil, sdkerrors.Wrap(types.ErrUnsupportedForContract, "fswap and fbridge not supported of Stargate")
```

Comment:
How about separating this error message (and `if`) for each module?

Reply:
https://github.com/Finschia/wasmd/commit/1ce21a05eac3d5288933e0395a44d4a38822b974 Updated, thank you.